wrap/python/gtsam_py/utils/test_case.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
this file, which I remember adding (and which has a correct docstring), makes me think these utils are the one we should keep.

Reply:
Yes, these are the ones I copied from `cython` because I think they are more tested than the versions in duy's fork...